test-cases/longevity/longevity-twcs-48h.yaml

```diff
@@ -10,14 +10,14 @@ stress_read_cmd: [
     ]
 
 
-n_db_nodes: 3
+n_db_nodes: 6
 n_loaders: 3
 n_monitor_nodes: 1
 simulated_racks: 3
 
 instance_type_db: 'i3en.2xlarge'
 
-nemesis_class_name: 'SisyphusMonkey'
+nemesis_class_name: 'RemoveNodeParallel'
```

Comment:
I would recommend doing testing changes in it's own commit 

also for those simple changes one can use `extra_environment_variables` pipeline parameter

```
SCT_TEST_DURATION=180
SCT_N_DB_NODES=9
SCT_NEMESIS_CLASS_NAME=RemoveNodeParallel
```

mixing it with the code under review is confusing (and can be merged in by mistake)